project-base/app/config/graphql/types/ModelType/Order/OrderItemConnection.types.yaml

```diff
@@ -0,0 +1,10 @@
+OrderItemConnection:
+    type: relay-connection
+    config:
+        nodeType: OrderItem
+        connectionFields:
+            totalCount:
+                type: "Int!"
+                description: "Total number of order items"
```

Comment:
probably latter since it works in decorator, updated
[fixup! fixup! fixup! order items search](https://github.com/shopsys/shopsys/pull/3295/commits/bcd9d70ec555cc0bfbfbd8ab09791d006f0d66f9)